lib/cli/index.js

```diff
@@ -1,9 +1,5 @@
 'use strict';
 
-// work around misbehaving libraries, so we can correctly cleanup before
-// actually exiting.
-require('capture-exit').captureExit();
-
```

Comment:
As far as I understand we actually use `capture-exit` to ensure that all exit handlers are finished.
If that is true seems like if we have no exit handlers we don't need a captured exit.

That's why I've moved `captureExit()` to `will-interrupt-process` utility which automatically captures exit once the first exit handler is registered.